### PR TITLE
Fix README for cassandra tool

### PR DESCRIPTION
**Pull request by vancexu**

### CODE REVIEW

File: tools/cassandra/README.md [hunk 1]
```diff
@@ -6,19 +6,26 @@ How
 ---
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
+- Make sure you already created cadence and visibility keyspaces (see `keyspace.cql` in schema)
 
 Setting up initial cassandra schema on a new cluster
 ----------------------------------------------------
 ```
 ./cadence-cassandra-tool -ep 127.0.0.1 -k cadence setup-schema -v 0.0 -- this sets up just the schema version tables with initial version of 0.0
```
Comment (madhuravi) [inline tools/cassandra/README.md]: There is a make target for this. You can just replace these with
make install-schema

File: tools/cassandra/README.md [hunk 2]
```diff
@@ -6,19 +6,26 @@ How
 ---
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
+- Make sure you already created cadence and visibility keyspaces (see `keyspace.cql` in schema)
```
Comment (madhuravi) [inline tools/cassandra/README.md]: This is a bit confusing. Aren't we creating this using the commands below? How will the user make sure they have already created the keyspaces.

File: tools/cassandra/README.md (hunk 1 above)
Comment (vancexu) [reply tools/cassandra/README.md]: make sense, added

File: tools/cassandra/README.md (hunk 2 above)
Comment (vancexu) [reply tools/cassandra/README.md]: removed

File: tools/cassandra/README.md [hunk 3]
```diff
@@ -7,18 +7,30 @@ How
...
 - You should see an executable `cadence-cassandra-tool`
 
-Setting up initial cassandra schema on a new cluster
+Setting up cassandra schema on a new cluster using Makefile
+----------------------------------------------------
+``` 
+make install-schema
+```
+
+Setting up schema on a new cluster manually
```
Comment (madhuravi) [inline tools/cassandra/README.md]: I don't think this section is necessary If folks want to dig in, they can easily look up Makefile to see command breakdown.

File: tools/cassandra/README.md [hunk 4]
```diff
@@ -7,18 +7,30 @@ How
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
 
-Setting up initial cassandra schema on a new cluster
+Setting up cassandra schema on a new cluster using Makefile
```
Comment (madhuravi) [inline tools/cassandra/README.md]: Can avoid "using Makefile". It's a bit confusing, Instead, you call it out as a shortcut.

**Pull request merged**